main_test.go

```diff
@@ -141,8 +187,9 @@ func TestClone(t *testing.T) {
 		b := &plugin.Build{Commit: c.commit, Branch: c.branch, Ref: c.ref, Event: c.event}
 		w := &plugin.Workspace{Path: dir}
 		v := &Params{
-			Recursive: recursive,
-			Tags:      tags,
+			Recursive:          recursive,
+			Tags:               tags,
+			TweakGitattributes: c.tweak,
 		}
 		if err := clone(r, b, w, v); err != nil {
 			t.Errorf("Expected successful clone. Got error. %s.", err)
```

Comment:
Would it also make sense to add a clause that reads the file contents not from filesystem but from git itself? The original motivation behind this change was that the file on filesystem would contain CLRF line endings, but the file in git did not. So would it make sense to check the file contents according to git and make sure that's consistent with what's in the filesystem?